Regulator_Linear.lib

```diff
@@ -4507,6 +4507,26 @@ X VOUT 5 300 100 100 L 50 50 1 1 w
 ENDDRAW
 ENDDEF
 #
+# TPS730285DBVT
```

Comment:
I still see a `T` here.

Reply:
Oh, my bad. Similar to the https://github.com/KiCad/kicad-symbols/pull/2515 pull request, I first created a standalone symbol and later realized that I could also create it as an alias of an already existing compatible symbol. I will delete this duplicate.